addons/web/static/src/search/control_panel/control_panel.xml

```diff
@@ -65,15 +65,15 @@
                     <div class="o_mobile_search_header">
                         <button
                             type="button"
-                            class="o_mobile_search_button btn"
+                            class="o_mobile_search_button btn btn-link text-reset"
```

Comment:
is adding `btn-link` and then  `text-reset` classes changing something compared to just keep the `btn` class?

Reply:
We can indeed remove the `btn-link`, and keep `btn` & `text-reset`-> white (1).	

With `btn btn-link` only -> cyan (2).

With `btn` only and no `text-reset` -> `#ccfffff` (3).

<img width="779" alt="image" src="https://user-images.githubusercontent.com/80678921/195075040-207d784c-de32-4160-aee8-4c5383935774.png">

@stefanorigano going with (1)?

Reply:
Thanks for the screenshoots.
**IF** light-grey links (3) become white on :hover let's go for that... otherwise let's go for 1.
Don't forget to check bright mode ;)

Reply:
We're on mobile here, there isn't really an hover state. There is the active state that I removed  here https://github.com/odoo/odoo/pull/102947#discussion_r992066160 because imo it didn't bring any value.

I could replicate what we see in discuss: a rgba .10 background ($o-black in darkmode/o-brand-odoo in light) if we want something like:

<img width="829" alt="image" src="https://user-images.githubusercontent.com/80678921/195079402-c556df8a-e2d0-4044-8e54-aeef31ef6e77.png">

Reply:
> We're on mobile here, there isn't really an hover state. 

Well, that change in bg-color after tapping it's a quite useful visual feedback ;)

> There is the active state that I removed here https://github.com/odoo/odoo/pull/102947#discussion_r992066160

Please don't , it's out of scope ;)